packages/react/src/components/hooks/usePermissions.ts

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from "react";
+import { Video } from "@signalwire/js";
+import jwt_decode from "jwt-decode";
+
+function makeBarePermissionObject(permString: string[]) {
```

Comment:
we are going to need a unit test for makeBarePermissionObject / decoratePermissionObject (or for `usePermissions` directly)